src/Collection/CollectionInterface.php

```diff
@@ -645,36 +642,36 @@ public function last();
      * Returns a new collection as the result of concatenating the list of elements
      * in this collection with the passed list of elements
      *
-     * @param iterable $items Items list.
+     * @param iterable $elements Items list.
```

Comment:
Isn't the term "a collection of items" more commonly used instead of "a collection of elements"?

Reply:
Documentation refers to the entries in the collection as "elements", but I agree that it's the wrong name for the parameter.

Since they can be collections themselves, either items or values makes sense to me. We can pick one and be consistent everywhere.

Reply:
Also we have methods `appendItem()` and `prependItem()`.

Reply:
I will switch the names to "items" and we can decide how to update documentation later.